src/scripts.js

```diff
@@ -153,6 +152,7 @@ function populateFriends() {
 }
 
 function applyFriendStyling() {
+  let friendsStepsParagraphs = document.querySelectorAll('.friends-steps');
```

Comment:
Interesting for dealing with selecting classes on elements which have been interpolated to the DOM